test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -60,6 +60,23 @@ class CheckYourAnswersControllerSpec extends SpecBase with BeforeAndAfterEach {
   val mockSessionRepository: SessionRepository = mock[SessionRepository]
   val mockAuditService : AuditService = mock[AuditService]
 
+  val safeID = Gen.oneOf(List("XADAC0000123456", "XE0001122334455", "XE0001234567890")).sample.get
```

Comment:
why are you using static values? there must be a generator already made. The point is we're explicitly not using identifiers in code.